litmus/sched_plugin.c

```diff
@@ -212,9 +213,22 @@ static DEFINE_RAW_SPINLOCK(sched_plugins_lock);
 	if (!plugin->func) \
 		plugin->func = litmus_dummy_ ## func;}
 
-/* FIXME: get reference to module  */
```

Comment:
This is the FIXME that I mean — the code should make sure that the module's reference count does not go to zero while the scheduler is in use, lest the active scheduler is being unloaded.

Reply:
Answered at the pull request conversation